scalding-core/src/main/scala/com/twitter/scalding/filecache/DistributedCacheFile.scala

```diff
@@ -96,9 +133,7 @@ object DistributedCacheFile {
 
 final case class UncachedFile private[scalding] (source: Either[String, URI]) {
 
-  import DistributedCacheFile._
-
-  def add()(implicit mode: Mode): CachedFile =
```

Comment:
it does not hurt compatibility because we don't return this class anywhere and he has the private constructor. Nobody can not use his directly.